plugin.xml

```diff
@@ -65,6 +65,8 @@
         <config-file target="AndroidManifest.xml" parent="application">
             <meta-data android:name="com.facebook.sdk.ApplicationId" android:value="@string/fb_app_id"/>
             <meta-data android:name="com.facebook.sdk.ApplicationName" android:value="@string/fb_app_name" />
+            <meta-data android:name="com.facebook.sdk.AdvertiserIDCollectionEnabled" android:value="false" />
+            <meta-data android:name="com.facebook.sdk.AutoLogAppEventsEnabled" android:value="false" />
```

Comment:
Note: cordova 7 will not include all `meta-data`.

May be using this is better:
```
      <custom-config-file parent="./application" target="AndroidManifest.xml">
    <meta-data android:name="com.facebook.sdk.AdvertiserIDCollectionEnabled" android:value="false" />
    <meta-data android:name="firebase_crash_collection_enabled" android:value="false" />
    <meta-data android:name="com.facebook.sdk.AutoLogAppEventsEnabled" android:value="false" />
</custom-config-file>
<plugin name="cordova-custom-config" spec="^5.1.0" />
```